src/modules/account/layouts/AccountLayout/AccountLayout.tsx

```diff
@@ -4,29 +4,41 @@ import { useRouter } from 'next/router';
 import { FC, useMemo } from 'react';
 
 import { ListNavigationItem } from '@shared/components';
+import { useHasAnyPermission } from '@shared/hooks/has-permission';
 import useTranslation from '@shared/hooks/use-translation/use-translation';
 import SidebarLayout from '@shared/layouts/SidebarLayout/SidebarLayout';
 
 import styles from './AccountLayout.module.scss';
 import { AccountLayoutProps } from './AccountLayout.types';
 
-import { ACCOUNT_NAVIGATION_LINKS } from 'modules/account/const';
+import { ACCOUNT_NAVIGATION_LINKS, Permission } from 'modules/account/const';
 
 const AccountLayout: FC<AccountLayoutProps> = ({ children, className, pageTitle }) => {
 	const { asPath } = useRouter();
+	const hasPerm = useHasAnyPermission(Permission.VIEW_OWN_MATERIAL_REQUESTS);
```

Comment:
```suggestion
	const hasMaterialRequestsPerm = useHasAnyPermission(Permission.VIEW_OWN_MATERIAL_REQUESTS);
```

maybe also build this PR on top of the other one, so you can already fix the merge conflicts